File: Week 1/2.py
# Find largest element in array 
class Solution:
    def largest(self, arr):
        return max(arr)

if __name__ == "__main__":
    # Taking array input from user
    arr_input = input("Enter the array elements separated by spaces: ")
    arr = list(map(int, arr_input.strip().split()))

    # Creating object and calling function
    solution = Solution()
    largest_element = solution.largest(arr)

    # Printing the result
    print("Largest element:", largest_element)

File: Week 1/2.c
#include <stdio.h>
int largest(int arr[], int n) {
    int mx = arr[0];
    for (int i = 0; i < n; i++) {
        if (mx < arr[i])
            mx = arr[i];
    }
    return mx;
}
int main() {
    int n;
    printf("Enter the number of elements in the array: ");
    scanf("%d", &n);
    int arr[n];
    // Taking array elements from user
    printf("Enter %d elements:\n", n);
    for (int i = 0; i < n; i++) {
        scanf("%d", &arr[i]);
    }
    // Finding and printing the largest element
    int result = largest(arr, n);
    printf("The largest element is: %d\n", result);
    return 0;
}
